gems/pending/appliance_console.rb

```diff
@@ -288,11 +289,16 @@ module ApplianceConsole
 
           if agree("Apply time and timezone configuration? (Y/N): ")
             say("Applying time and timezone configuration...")
-            Env['TIMEZONE'] = new_loc, new_city
-            Env['TIME'] = new_date, new_time unless Env.error?
+            t = Time.new(*new_date.split('-'), *new_time.split(':'))
+            begin
+              LinuxAdmin::TimeDate.set_system_timezone(new_loc, new_city)
+              LinuxAdmin::TimeDate.set_system_time(t)
+            rescue AwesomeSpawn::CommandResultError => e
```

Comment:
This now feels weird to catch AwesomeSpawn from LinuxAdmin.  I feel like either LinuxAdmin should expose its own exception, or we just capture StandardError here: `rescue => e`